domains/certificates/Query.API/API.IntegrationTests/CertificatesByOwnerProjectionTest.Apply_TwoCreatedEvents_HasTwoCreatingCertificates.verified.txt

```diff
@@ -15,7 +15,7 @@
     Guid_3: {
       CertificateId: Guid_3,
       DateFrom: 123,
-      DateTo: 123,
+      DateTo: 124,
```

Comment:
Because the test used a period that was from 123 to 123, so that is a period of length 0. In this PR the Period valueobject only allows a non-zero period. That is why the test must be changed in order to have a valid period (with a length larger than 0).